### DISCUSSION TITLE
Ability to shoot fireball and customize fireball knockback

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#123] [To-do List]
Status: Not started (5/2/2023), Finished (8/3/2023)

**Comment by HierarchyMan** [comment]
TNT too?
And is it gonna be auto enabled with bed gamerule or r u gonna add a new gamerule?

**Reply by GoodestEnglish** [reply]
It will be auto enabled. Not sure about TNT but I will see what I can do with it.

**Reply by HierarchyMan** [reply]
Hopefully there will be an option to customise block whitelist for tnt and fireball seperately

**Comment by GoodestEnglish** [comment]
For more info: This will be the LAST update before the official release (aka production ready). Please report any issues and bugs if you encounter those.
